caikit_nlp/modules/token_classification/filtered_span_classification.py

```diff
@@ -173,16 +172,15 @@ def run(
                         token_classification_results.append(token_classification)
         return TokenClassificationResult(results=token_classification_results)
 
-    @TokenClassificationTask.taskmethod(input_streaming=True, output_streaming=True)
     def run_bidi_stream(
-        self, text: Iterable[str], threshold: Optional[float] = None
+        self, text_stream: Iterable[str], threshold: Optional[float] = None
     ) -> Iterable[StreamingTokenClassificationResult]:
         """Run bi-directional streaming inferencing for this module.
         Run classification on text split into spans. Returns results
         based on score threshold for labels that are to be outputted
 
         Args:
-            text: Iterable[str]
+            text_stream: Iterable[str]
```

Comment:
if we're keeping this we should update the task's streaming output and probably keep the decorators here